src/lib/components/form/BuildForm.svelte

```diff
@@ -356,6 +363,12 @@
           filtered={filteredItems}
         />
       {/if}
+
+      {#if validations[`items-length-${currentRoundIndex}`] === false}
+        <div class="form-text-error" transition:slide={{ duration: 100 }}>
+          You have selected more than 6 items for this round.
```

Comment:
```suggestion
          You would end up with more than 6 items this round.
```
Technically you could select 5 items in the previous round, then select 2 more and be over the limit. I think this wording works better!

Reply:
Sure! I was having trouble wording this correctly, your suggestion makes sense